modules/installation-nutanix-config-yaml.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/configuring-iam-nutanix.adoc
+
+:_content-type: REFERENCE
+[id="installation-nutanix-config-yaml_{context}"]
+= Sample customized install-config.yaml file for Nutanix
+
+You can customize the `install-config.yaml` file to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your `install-config.yaml` file by using the installation program and modify it.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+compute: <2> <3>
+- hyperthreading: Enabled <4>
+  name: worker
+  platform: {}
+  replicas: 3
+controlPlane: <2> <3>
+  hyperthreading: Enabled <4>
+  name: master
+  platform: {}
+  replicas: 3
```

Comment:
I think this should resemble the [vSphere platform config](https://docs.openshift.com/container-platform/4.10/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html#installation-installer-provisioned-vsphere-config-yaml_installing-vsphere-installer-provisioned-customizations).  The only deviations are memoryMB and diskSizeGB parameters.  For Nutanix, these parameters are [memoryMiB and diskSizeMiB](https://github.com/openshift/installer/blob/master/pkg/types/nutanix/machinepool.go#L5-L26) respectively.

Reply:
Thanks for catching this. I could have sworn I based the Nutanix sample on the vSphere sample, but clearly I didn't.

Reply:
Done.